src/Compilers/CSharp/Portable/Emitter/Model/PEModuleBuilder.cs

```diff
@@ -40,6 +40,28 @@ public override NoPia.EmbeddedTypesManager EmbeddedTypesManagerOpt
         /// </summary>
         private Dictionary<FieldSymbol, NamedTypeSymbol> _fixedImplementationTypes;
 
+        private bool _needsGeneratedIsReadOnlyAttribute_IsFrozen;
+        private bool _needsGeneratedIsReadOnlyAttribute_Value;
```

Comment:
@cston Can you please clarify? it sounds less clear to me to use `Unknown` to indicate that it is for example sealed, or set but not sealed yet.

---
In reply to: [115029940](https://github.com/dotnet/roslyn/pull/18715#discussion_r115029940) [](ancestors = 115029940)